src/content/1.7/scale/optimizations.md

```diff
@@ -141,31 +159,59 @@ If you're using PHP-FPM, you should be able to use apache mpm_event. Using the f
 
 ### 5) MySQL/MariaDB Settings
 
-If you using MySQL < 8 or MariaDB, enable the query cache by putting this setting in the ```/etc/mysql/my.cnf``` file:
+Though we’re discussing it just now, MySQL tuning is just as important.
+
+Our intention here is to optimize the instance’s throughput by adding caches.
+
+As for PHP, it allows the service to work as much as possible in memory and avoid disk accesses, hence reducing latency.
+
+#### Caching
+
+Those parameters allow better cache information for further reuse, first by enabling it, then by increasing its sizes. 
+Again, the idea is to keep the query results in memory rather than looking them up to the (higher latency) hard drive.
+
+As always, those values should be adapted to your own environment, you probably won’t need a ``host_cache_size`` of 1000. 
 
 ```
-query_cache_limit               = 128K
-query_cache_size                = 32M
-query_cache_type                = ON
+query_cache_limit = 128K 
+query_cache_size = 32M
+query_cache_type = ON
+table_open_cache = 1000
+thread_cache_size = 80
+host_cache_size=1000
 ```
 
-Other important settings are:
+#### Buffering
+
+Buffering is almost another word for caching. 
+
+So we work here with the memory area that holds cached data for InnoDB tables, indexes, and other auxiliary buffers, etc..
+
+Again, those options should be adapted to your shop. 
+
+Setting up the ```innodb_buffer_pool_size``` to 1G maybe too much for your SQL instance, just make sure you have enough memory according to the value you configure.
+
+The important thing is, if possible, to set ```innodb_buffer_pool_size``` to a greater value than your database size.
+
 
 ```
-table_open_cache                      = 1000
-read_buffer_size                      = 2M
-read_rnd_buffer_size                  = 1M
-thread_cache_size                     = 80
-join_buffer_size                      = 2M
-sort_buffer_size                      = 2M
-max_connections                       = 400
-tmp_table_size                        = 32M
-max_heap_table_size                   = 32M
-table_definition_cache                = 1000
-performance_schema                    = OFF
+read_buffer_size			= 2M 
+read_rnd_buffer_size			= 1M
+join_buffer_size			= 2M 
+sort_buffer_size 			= 2M
+innodb_buffer_pool_size 		= 1G
+
```

Comment:
```suggestion
read_buffer_size			= 2M 
read_rnd_buffer_size		= 1M
join_buffer_size			= 2M 
sort_buffer_size 			= 2M
innodb_buffer_pool_size 	= 1G

```

Reply:
Strange, yours are not aligned anymore Oo
Am I missing anything ?

Reply:
It looks well on my end 🤔 

![Screenshot 2020-04-22 at 16 48 55](https://user-images.githubusercontent.com/1009343/79996839-31f24a80-84b9-11ea-951c-00b4d25bf5ad.png)

Reply:
That's a funny one ^^

![Capture d’écran 2020-04-23 à 00 51 17](https://user-images.githubusercontent.com/16487473/80041373-997db980-84fc-11ea-8e98-3af52398f611.png)